src/components/Dialogs/ImageCropDialog/cropper.ts

```diff
@@ -53,8 +53,12 @@ export const useCropper = ({ imageEl, imageType }: UseCropperOpts) => {
   const zoomStep = (zoomRange[1] - zoomRange[0]) / 20
 
   const handleZoomChange = (zoom: number) => {
-    setCurrentZoom(zoom)
-    cropper?.zoomTo(zoom)
+    const [minZoom, maxZoom] = zoomRange
+    const isInRange = zoom >= minZoom - zoomStep && zoom <= maxZoom + zoomStep
+    if (isInRange) {
+      setCurrentZoom(zoom)
+      cropper?.zoomTo(zoom)
+    }
```

Comment:
I think what could be helpful here is that the value exceeds the range, just make it min/max from that range, the we can avoid the situations where the user is less than `step` from min/max but cannot move to min/max itself. Does that make sense?